test/unit/dom/dom.js

```diff
@@ -200,9 +588,33 @@ describe('skatejs-named-slots dom', function () {
 
         host.innerHTML = processingInstruction;
         expect([processingInstruction, processingInstructionsAfterInnerHtml].indexOf(host.innerHTML)).to.be.above(-1);
+        expect(host.childNodes.length).to.equal(1);
+        expect(slot.getAssignedNodes().length).to.equal(1);
```

Comment:
We shouldn't need to test for projection of anything but `Text` and `Element` nodes as per the spec http://w3c.github.io/webcomponents/spec/shadow/#dfn-slot-assignable. These should still be "light DOM" and be reported as `childNodes` of the host element, but not assigned to a slot.